cinema/serializers.py

```diff
@@ -1 +1,90 @@
-# write serializers here
+from rest_framework import serializers
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.ReadOnlyField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row", "capacity"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    genres = GenreSerializer(many=True, read_only=True)
+    actors = ActorSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieSessionSerializer(serializers.ModelSerializer):
+    movie_title = serializers.SerializerMethodField()
+    cinema_hall_name = serializers.SerializerMethodField()
+    cinema_hall_capacity = serializers.SerializerMethodField()
+
+    class Meta:
+        model = MovieSession
+        fields = [
+            "id",
+            "show_time",
+            "movie_title",
+            "cinema_hall_name",
+            "cinema_hall_capacity"
+        ]
+
+    def get_movie_title(self, obj):
+        return obj.movie.title
+
+    def get_cinema_hall_name(self, obj):
+        return obj.cinema_hall.name
+
+    def get_cinema_hall_capacity(self, obj):
+        return obj.cinema_hall.capacity
+
+
+class MovieListSerializer(serializers.ModelSerializer):
+    genres = serializers.StringRelatedField(many=True)
+    actors = serializers.SerializerMethodField()
```

Comment:
why not using the same way? StringRelatedField for both fields or SerializerMethodField for both fields

Reply:
in the first case I use a string, and in the second method get_actors